gitaware.go

```diff
@@ -17,10 +17,10 @@ func (c *PushMetadataPlugin) Run(cliConnection plugin.CliConnection, args []stri
 	if args[0] != pluginCommand {
 		return
 	}
-	output, _ := sh.Command("git", "rev-parse", "HEAD").Output()
-	fmt.Println("SHA" + string(output))
+	output, _ := metadata.GetRevision(".")
```

Comment:
you might want to handle the error. in case something happens like someone happens to run `git-push` not in the folder of a git repo. maybe you still want to push (and skip writing the file) or just flat out abort?